libraries-bytecode/src/main/java/com/baeldung/incompatibleclasschange/libraries/modified/Dinosaur.java

```diff
@@ -0,0 +1,11 @@
+package com.baeldung.incompatibleclasschange.libraries.modified;
+
+public class Dinosaur {
+    public static void species(String sp) {
+	if(sp == null) {
+		System.out.println("I am a generic Dinosaur");
```

Comment:
The spacing in this file is incorrect. It uses tabs, where we should be using spaces.

Reply:
Fixed.